### DISCUSSION TITLE
Gravatar and Theme Environment Variable

### DISCUSSION THREAD

**Discussion by root9191** [dani-garcia/vaultwarden#2401] [Q&A]
Hello,
are there Environment Variables for Enabling Gravar and Dark Theme by Default?

**Comment by Masgalor** [comment]
There are no serverside configurations for the webvault (to my knowledge).

If you want to change something you need to build it with those options or modify the source.
Regarding the dark theme see  #2220

**Reply by BlackDex** [reply]
Also, all these settings are basically user options which are stored within cookies/local-storage of the browser.
These are not stored within a profile at the server side.
